db/migrate/20200408050542_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.text :body
+      t.string :tone
```

Comment:
@d-atkins @jklich151  @mronauli  
I made the body a text data type because of the length we discussed. Should it be switched to string or something else? Also, is the tone supposed to be a string or some other data type?

Reply:
@eamouzou @d-atkins @jklich151 i think text data type makes sense given how many characters we'll limit to a post. also, if we stick to a post having multiple moods how about an array?